src/clients/sentinel.ts

```diff
@@ -78,5 +83,13 @@ export class Sentinel extends Single {
             })
             .connect();
     }
+
+    async quit() {
+        await super.quit();
+        if (this.sentinelClient) {
+            const reply = this.sentinelClient.quit();
+            return reply.then(() => {})
```

Comment:
why no
```
await this.sentinelClient.quit()
```

Reply:
cause that won't compile due to the Promise signature